telemetry/definitions/commonDefinitions.json

```diff
@@ -3146,6 +3152,24 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_humanInTheLoop",
```

Comment:
can this be a field on an existing metric? it's not clear from the description what its purpose is